client/src/main/java/io/pravega/client/segment/impl/SegmentOutputStreamImpl.java

```diff
@@ -651,8 +657,18 @@ void reconnect() {
                               state.failConnection(e1);
                               throw Exceptions.sneakyThrow(e1);
                           }
+                          //add timeout and call fail and throw.
+                          //a late timeout if fine it will just cause a spurious connection close.
```

Comment:
`if fine` -> `is fine`? Also, please lines should start with a space and capital letter.